app.py

```
@@ -114,68 +93,74 @@ def get_details():
+        template = "logged_out.html"
 
     return render_template(template, **data)
 
 @app.route("/api_demo")
 def get_api_demo():
     template = "api_demo.html"
 
-    if session.get("user"):
-        kinde_client = user_clients.get(session.get("user"))
-        data = {"current_year": date.today().year}
+    kinde_client = user_clients.get(session.get("user"))
```

Ruff (0.11.9)

Undefined name `user_clients`

(F821)